scripts/e2e_get_tools.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -o errexit
+# ensure this file is sourced to add required components to PATH
+
+here="$(dirname "$(readlink --canonicalize "${BASH_SOURCE[0]}")")"
+root="$(readlink --canonicalize "$here/..")"
+
+VERSION="v0.11.1"
+KIND_BINARY_URL="https://github.com/kubernetes-sigs/kind/releases/download/${VERSION}/kind-$(uname)-amd64"
+K8_STABLE_RELEASE_URL="https://storage.googleapis.com/kubernetes-release/release/stable.txt"
+
+if [ ! -d "${root}/bin" ]; then
+    mkdir "${root}/bin"
+fi
+
+echo "retrieving kind"
+curl --max-time 10 --retry 10 --retry-delay 5 --retry-max-time 60 -Lo "${root}/bin/kind" "${KIND_BINARY_URL}"
```

Comment:
can you remove (or increase) the `--max-time` parameters? I'm currently download kind binaries in ~25s